aws-py-static-website/README.md

```diff
@@ -0,0 +1,138 @@
+    +   │  ├─ aws:s3:BucketObject         404.html                                  create
+    +   │  └─ aws:s3:BucketObject         index.html                                create
+    +   ├─ aws:acm:Certificate            certificate                               create
+    +   ├─ aws:route53:Record             ***-validation                            create
+    +   ├─ aws:acm:CertificateValidation  certificateValidation                     create
+    +   ├─ aws:cloudfront:Distribution    cdn                                       create
+    +   └─ aws:route53:Record             ***                                       create
+    ```
+
+1.  To see the resources that were created, run `pulumi stack output`:
+
+    ```bash
+    $ pulumi stack output
+    Current stack outputs (4):
+        OUTPUT                           VALUE
+        cloudfront_domain                ***.cloudfront.net
+        content_bucket_url               s3://***
+        content_bucket_website_endpoint  ***.s3-website-us-west-2.amazonaws.com
+        target_domain_endpoint           https://***/
+    ```
+
+1.  To see that the S3 objects exist, you can either use the AWS Console or the AWS CLI:
+
+    ```bash
+    $ aws s3 ls $(pulumi stack output content_bucket_url)
+    2020-02-21 16:58:48        262 404.html
+    2020-02-21 16:58:48        394 index.html
+    ```
+
+1.  Open a browser to the target domain endpoint from above to see your beautiful static website. (Since we don't wait for the CloudFront distribution to completely sync, you may have to wait a few minutes)
+
+1.  To clean up resources, run `pulumi destroy` and answer the confirmation question at the prompt.
+
+## Troubleshooting
+
+### Scary HTTPS Warning
+
+When you create an S3 bucket and CloudFront distribution shortly after one another, you'll see
+what looks to be HTTPS configuration issues. This has to do with the replication delay between
+S3, CloudFront, and the world-wide DNS system.
+
+Just wait 15 minutes or so, and the error will go away. Be sure to refresh in an incognito
+window, which will avoid any local caches your browser might have.
+
+### "PreconditionFailed: The request failed because it didn't meet the preconditions"
+
+Sometimes updating the CloudFront distribution will fail with:
+
+```text
+"PreconditionFailed: The request failed because it didn't meet the preconditions in one or more
+request-header fields."
+```
+
+This is caused by CloudFront confirming the ETag of the resource before applying any updates.
+ETag is essentially a "version", and AWS is rejecting any requests that are trying to update
+any version but the "latest".
+
+This error will occurr when the state of the ETag get out of sync between the Pulumi Service
+and AWS. (Which can happen when inspecting the CloudFront distribution in the AWS console.)
+
+This will get fixed in Pulumi soon, but for the time being you can find workaround steps in
```

Comment:
We now have the ability to run a `pulumi refresh` to pickup any newer ETag values. So we should update the text to just say that instead of indirecting through a GitHub issue. (Would you mind updating the `README.md` file for the TypeScript version of this example to use the same text, too?)